kernel/ukvm/kernel.c

```diff
@@ -42,5 +42,5 @@ void _start(void *arg)
     ret = solo5_app_main(cmdline);
     log(DEBUG, "Solo5: solo5_app_main() returned with %d\n", ret);
 
-    platform_exit();
+    platform_exit(ret);
```

Comment:
I think this should be `solo5_exit(ret)` for consistency, also in the virtio version.

Reply:
changed on both ukvm and virtio.